templates/plugin/container_gateway.yml.erb

```diff
@@ -6,3 +6,24 @@
 <% if scope.lookupvar("foreman_proxy::plugin::container_gateway::sqlite_timeout") -%>
 :sqlite_timeout: <%= scope.lookupvar("foreman_proxy::plugin::container_gateway::sqlite_timeout") %>
 <% end -%>
+<% if scope.lookupvar('foreman_proxy::plugin::container_gateway::database_backend') == 'postgres' -%>
+:db_connection_string: <%= [
```

Comment:
Would it make more sense to use `URI` to do the heavy lifting?
```erb
<%-
case scope.lookupvar('foreman_proxy::plugin::container_gateway::database_backend')
when 'postgresql'
  uri = URI("postgresql://")
  uri.user = scope.lookupvar('foreman_proxy::plugin::container_gateway::postgresql_user')
  uri.password = scope.lookupvar('foreman_proxy::plugin::container_gateway::postgresql_user')
  uri.host = scope.lookupvar('foreman_proxy::plugin::container_gateway::postgresql_host')
  uri.port = scope.lookupvar('foreman_proxy::plugin::container_gateway::postgresql_port')
  uri.path = scope.lookupvar('foreman_proxy::plugin::container_gateway::postgresql_database')
when 'sqlite'
  uri = URI("sqlite://#{scope.lookupvar("foreman_proxy::plugin::container_gateway::sqlite_db_path")}")
end
-%>
:db_connection_string: <%= uri %>
```

Reply:
nice, :+1: from me